### ISSUE TITLE
now bot not wroking

### ISSUE THREAD

**Issue by Samiran446** [ymmmmmmmm/getgrass_bot#16]
After yesterday's maintenance, using Sk5 connection all show ERROR | main:connect_to_wss:70 - Connection not allowed by ruleset

**Comment by moorejen** [comment]
did u find a solution?

**Comment by Samiran4461** [comment]
No I already contacted dev but he say it's working 😕 but really it's not working

**Comment by azamat5678** [comment]
Yes, it's not working right now. There have been changes on the site

**Comment by Samiran4461** [comment]
Give me your contact tg

**Comment by youngbloodes** [comment]
Same problem, did you find a fix?
